### PR TITLE
Automated cherry pick of #75073: Allow session affinity a period of time to setup for new

### PR DESCRIPTION
**Pull request by grayluck**
Cherry pick of #75073 on release-1.13.

#75073: Allow session affinity a period of time to setup for new

### CODE REVIEW

**Review by MrHohn — APPROVED** [review]
/ok-to-test
/lgtm
/approve

**Comment by grayluck** [comment]
Thanks MrHohn :D
Assigning patch release manager for cherrypick approval.
/assign @aleksandra-malinowska @tpepper

**Comment by schoolteacher1** [comment]
Life is good

**Comment by mariantalla** [comment]
`pull-kubernetes-integration`'s failed run seems to be a flake (related: https://github.com/kubernetes/kubernetes/issues/72493)

/retest

**Comment by mariantalla** [comment]
(talking about [#48680](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/75341/pull-kubernetes-integration/48680/))

**Comment by mariantalla** [comment]
@aleksandra-malinowska / @tpepper , are you happy to merge this to 1.13? 🙏

**Comment by mariantalla** [comment]
@grayluck it looks like the PR also needs a release note (it could be `none` if you feel it's not applicable, I'll leave that to you)

**Comment by MrHohn** [comment]
(Checking if this still work.)
/release-note-none

**Pull request merged**
